source/jormungandr/jormungandr/instance.py

```diff
@@ -333,6 +343,14 @@ def get_realtime_proxies_from_db(self):
         result = models.external_services if models else None
         return [res for res in result if res.navitia_service == 'realtime_proxies']
 
+    def get_bss_stations_from_db(self):
+        """
+        :return: a callable query of external services associated to the current instance in db
+        """
+        models = self._get_models()
+        result = models.external_services if models else None
+        return [res for res in result if res.navitia_service == 'bss_stations']
```

Comment:
What if `models is None`? We would iterate on a non-iterable type, I think:
```suggestion
        result = models.external_services if models else []
        return [res for res in result if res.navitia_service == 'bss_stations']
```

Reply:
:+1: